src/scrabble.js

```diff
@@ -1,5 +1,165 @@
-function scrabble() {
-  // write code here
+const scoreLibrary = [
+  { letters: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'], score: 1 },
+  { letters: ['D', 'G'], score: 2 },
+  { letters: ['B', 'C', 'M', 'P'], score: 3 },
+  { letters: ['F', 'H', 'V', 'W', 'Y'], score: 4 },
+  { letters: ['K'], score: 5 },
+  { letters: ['J', 'X'], score: 8 },
+  { letters: ['Q', 'Z'], score: 10 }
+]
+
+const scrabble = (str = '') => {
+  if (typeof str !== 'string') return 0
+
+  let scores = 0
+
+  const splitStr = str.split('')
+
+  if (
+    (splitStr.includes('{') && splitStr.includes('}')) ||
+    (splitStr.includes('[') && splitStr.includes(']')) ||
+    /^[A-Z]*$/.test(str.toUpperCase())
+  ) {
+    let newStr = str
+
+    // MULTIPLE BRACKETS
```

Comment:
Loved how you're using comments to breakdown sections - really easy to read and navigate towards

Reply:
Thanks :)